utilities/chemiscope.py

```diff
@@ -91,27 +91,23 @@ def chemiscope_input(meta, frames, projection, prediction,
 
     .. _`ase.Atoms`: https://wiki.fysik.dtu.dk/ase/ase/atoms.html
     '''
-
     data = {
         'meta': {
             'name': meta['name'],
-            'description': meta.get('description', None),
-            'authors': meta.get('authors', None),
-            'references': meta.get('references', None),
+            'description': meta.get('description', " "),
+            'authors': meta.get('authors', []),
+            'references': meta.get('references', []),
         }
```

Comment:
@Luthaf will this prevent a necessary error message?

Reply:
No, but this is adding data where there is none, and we loose the distinction between "there is no reference information in this file" and "there are no references associated with this file", which I would like to preserve if possible. 

Chemiscope should be fine with `undefined` here, my guess with this issue is that the Python json module transforms `None` into `null`, which is different from `undefined`. There should be a way of dealing with this while keeping the None/`[]` separation.